requirements.txt

```diff
@@ -2,7 +2,7 @@
 FastChat
 GitPython>=3.1.42,<4.0.0
 shortuuid
-openai<1.0.0
```

Comment:
are there any implications to this bump? or is it a clean reversioning?

Reply:
I might not be understanding the questions.  But there were several changes with the API that needed corresponding changes on our side that are in the commit.